sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -4526,8 +4612,8 @@ enqueueMemFillHelper(pi_command_type CommandType, pi_queue Queue, void *Ptr,
   // Get a new command list to be used on this call
   ze_command_list_handle_t ZeCommandList = nullptr;
   ze_fence_handle_t ZeFence = nullptr;
-  if (auto Res = Queue->Context->getAvailableCommandList(Queue, &ZeCommandList,
-                                                         &ZeFence))
+  if (auto Res = Queue->Context->getAvailableCommandList(
+          Queue, &ZeCommandList, &ZeFence, true /* IsCopyCommand */))
```

Comment:
Hmm, I wonder if a memory fill will really benefit from using a copy engine.

Reply:
Specification says copy engine can handle fill operations.

Reply:
My comment wasn't about correctness, but whether it made sense from a performance point of view.

Reply:
Added a comment about perf improvements by using blitter

Reply:
I added a comment about my performance analysis runs for memory fill operation